pkg/scheduler/framework/v1alpha1/types_test.go

```diff
@@ -630,7 +630,7 @@ func TestNodeInfoAddPod(t *testing.T) {
 		},
 		NonZeroRequested: &Resource{
 			MilliCPU:         1300,
-			Memory:           209716200, //200MB + 1000 specified in requests/overhead
+			Memory:           1000,
```

Comment:
why are the 200MB going away?

Reply:
Because the return value `GetResourceRequest` is a non-zero value, we shouldn't consider the default value for resource requests.

The default value of resource requests we want to give to pod should be assigned when the return value `GetResourceRequest` is zero. Right?

Reply:
Perhaps we can do this in 1.19+, but we shouldn't change the behavior in older versions.
Please focus on fixing the bug in this PR so that we can cherrypick ASAP.